### ISSUE TITLE
License and modpack permissions?

### ISSUE THREAD

**Issue by aionys** [QMXTech/MachineMusePowersuits#21]
MachineMuse has explicitly stated the PowerSuits mod is allowed in modpacks, and Andrew has said pretty much the same.

Are you sticking with that decision wrt your fork of the mods?

**Comment by Korynkai** [comment]
Of course! As stated in #18, the license has not changed and continues to be a BSD 2-clause variant. I have no plans to change this, now or ever, which means anybody can do virtually whatever they want with the source, binaries and resources, so long as they comply with the following terms (as stated in the file LICENSE):

>Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:

>1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

>2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.

>Neither the name(s) of the author(s) nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
